templates/ff.config.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Let's remove everything in the templates dir or gitignore it. I recommend the latter if you use any code generator.